schemas/dataset-schema.json

```diff
@@ -269,15 +284,28 @@
         "properties": {
           "title": {
             "type": "string",
-            "title": "Title of this ETL table",
-            "description": "Title of the table (mostly for internal purposes, or for users of our data catalog) which is a few words description of the table (follow the guidelines as origin's `title`). NOTE: Table titles should be propagated automatically from snapshots (from origin's `title`). But, if there are multiple tables, or if the user wants to manually edit the title, it can be done by editing `table.title`.",
-            "requirement_level": "required (often automatic)"
+            "title": "Title of the ETL table",
+            "description": "Title of the table (mostly for internal purposes, or for users of our data catalog) which is a few words description of the table. NOTE: Table titles should be propagated automatically from snapshots (from origin's `title`). But, if there are multiple tables, or if the user wants to manually edit the title, it can be done by editing `table.title`.",
+            "requirement_level": "required (often automatic)",
+            "guidelines": [
+              ["Must start with a capital letter."],
+              ["Must not end with a period."],
+              ["Should identify the table."],
+              ["If the table has only one origin, use the title of the data product or snapshot."]
+            ]
           },
           "description": {
             "type": "string",
             "title": "Description of this ETL table",
-            "description": "Description of the table (mostly for internal purposes, or for users of our data catalog) which is a one- (or a few) paragraph description of the table (follow the guidelines of origin's `description`). NOTE: Table descriptions should be propagated automatically from snapshots (from origin's `description`). But, if there are multiple tables, or if the user wants to manually edit the description, it can be done by editing `table.description`.",
-            "requirement_level": "recommended (often automatic)"
+            "description": "Description of the table (mostly for internal purposes, or for users of our data catalog) which is a one- (or a few) paragraph description of the table. NOTE: Table descriptions should be propagated automatically from snapshots (from origin's `description`). But, if there are multiple tables, or if the user wants to manually edit the description, it can be done by editing `table.description`.",
```

Comment:
same comments as in `tables.title` (e.g. breakline and referring to the field as this field).